docs/guides/error-warnings-details.md

```diff
@@ -147,32 +147,32 @@ Error:
 - `"whatis" is not supplied OR doesn\'t have a case yet! [E013]`
 
 Details:  
-An Value not specified by the enum `WhatIsIt` was provided (no case matched), [read more about it here](../api/decorators/prop.md#whatisit)
+A Value not specified by the enum `WhatIsIt` was provided (no case matched), [read more about it here](../api/decorators/prop.md#whatisit)
 
 ### Input was not string or have .typegooseName function/string [E014]
 
 Error: `Input was not a string AND didnt have a .typegooseName function AND didnt have a .typegooseName string [E014]`
 
 Details:  
-The Provided Input wasnt an string and didnt have an `.typegooseName` function / string to be searched by
+The Provided Input wasnt a string and didnt have a `.typegooseName` function / string to be searched by
 
 ### customName must be string and at least one character [E015]
 
 Error: `"customName" must be a string AND at least one character ("${baseName}") [E015]`
 
 Details:  
-The `customName` option must be an String AND at least *one* character long
+The `customName` option must be a String AND at least *one* character long
 
 ### Type dosnt have "OptionsConstructor" [E016]
 
-Error: `Type does not have an valid "OptionsConstructor"! (${getName(loggerType)} on ${getName(target)}.${pkey}) [E016]`
+Error: `Type does not have a valid "OptionsConstructor"! (${getName(loggerType)} on ${getName(target)}.${pkey}) [E016]`
```

Comment:
thanks for the correction, but this is showing the line that gets output, so it would also needed to be changed in the code